src/modules/Dashboard/features/Managers/Popups/ManagersRemoveAccessPopup/ManagersRemoveAccessPopup.test.tsx

```diff
@@ -1,12 +1,14 @@
-import { waitFor, screen } from '@testing-library/react';
+import { waitFor, screen, fireEvent } from '@testing-library/react';
 import Router from 'react-router';
+import axios from 'axios';
 
 import { renderWithProviders } from 'shared/utils/renderWithProviders';
 
-import { ManagersRemoveAccessPopup } from './ManagersRemoveAccessPopup';
+import { ManagersRemoveAccessPopup } from '.';
 
 const refetchManagersMock = jest.fn();
 const onCloseMock = jest.fn();
+const fakeRequest = () => new Promise((res) => res(null));
```

Comment:
let's move such functions in shared/utils/tests or so. Could you please check LinkGenerator.test.tsx, DeletePopup.test.tsx as well